hydroengine_service/main.py

```diff
@@ -1240,22 +1237,76 @@ def api_get_raster():
     data = {'url': url}
     return Response(json.dumps(data), status=200, mimetype='application/json')
 
-
-@v1.route('/get_liwo_scenarios', methods=['GET', 'POST'])
+@v2.route('/get_liwo_scenarios', methods=['GET', 'POST'])
 @flask_cors.cross_origin()
 def get_liwo_scenarios():
     r = request.get_json()
 
-    # Currently 'liwo' only option
-    variable = 'liwo'
     # name of breach location as string
     liwo_ids = r['liwo_ids']
     # band name as string
     band = r['band']
 
-    raster_assets = {
-        'liwo': 'users/rogersckw9/liwo/liwo-scenarios-03-2019'
+    collection = 'projects/deltares-rws/liwo/production'
+    id_key = 'Scenario_ID'
+    bands = {
+        'waterdepth': 'waterdiepte',
+        'velocity': 'stroomsnelheid',
+        'riserate': 'stijgsnelheid',
+        'damage': 'schade',
+        'fatalities': 'slachtoffers',
+        'affected': 'getroffenen',
+        'arrivaltime': 'aankomsttijd'
     }
+    reducers = {
+        "waterdepth": "max",
+        "velocity": "max",
+        "riserate": "max",
+        "damage": "max",
+        "fatalities": "max",
+        "affected": "max",
+        "arrivaltime": "min"
+    }
+    # TODO: do we also map old band names for new collection?
+    assert band in bands
+    band_name = bands[band]
+    reducer = reducers[band]
+
+    image = liwo_functions.filter_liwo_collection_v2(collection, id_key, liwo_ids, band_name, reducer)
+    params = liwo_functions.get_liwo_styling(band)
+    info = liwo_functions.generate_image_info(image, params)
+    info['liwo_ids'] = liwo_ids
+    info['band'] = band
+
+    # Following needed for export:
+    # Specify region over which to compute
+    # export  is True or None/False
+    if r.get('export'):
+        region = ee.Geometry(r['region'])
+        # scale of pixels for export, in meters
+        info['scale'] = float(r['scale'])
+        # coordinate system for export projection
+        info['crs'] = r['crs']
+        extra_info = liwo_functions.export_image_response(image, region, info)
+        info.update(extra_info)
+
+    return Response(
+        json.dumps(info),
+        status=200,
+        mimetype='application/json'
+    )
+
+@v1.route('/get_liwo_scenarios', methods=['GET', 'POST'])
+@flask_cors.cross_origin()
+def get_liwo_scenarios():
+    r = request.get_json()
+    # name of breach location as string
+    liwo_ids = r['liwo_ids']
+    # band name as string
+    band = r['band']
+
+    collection = 'users/rogersckw9/liwo/liwo-scenarios-03-2019'
```

Comment:
for  other datasets this was in config.

Reply:
Since we style by sld styling it doesn't really fit with the other dataset vis definitions, which is why I've kept it separate for now.